### PR TITLE
Use eslint-config-standard

### PR DESCRIPTION
**Pull request by limonte**
Related to https://github.com/sweetalert2/sweetalert2/pull/1641#issuecomment-508757657

### CODE REVIEW

File: test/qunit/tests.js [hunk 1]
```diff
@@ -677,10 +677,10 @@ QUnit.test('backdrop accepts css background param', (assert) => {
   })
   assert.notOk(Swal.getContainer().style.background)
 
-  let backdrop = 'rgb(123, 123, 123)'
+  const backdrop = 'rgb(123, 123, 123)'
   Swal.fire({
     title: 'I have a custom backdrop',
-    backdrop: backdrop
+    backdrop
```
[the remark this answers is not in the document]
Comment (limonte) [reply test/qunit/tests.js]: It is correct, it's the new ES6 feature: https://alligator.io/js/object-property-shorthand-es6/

But, if it brings the readability issues, I'll roll it back :+1:

[the remark this answers is not in the document]
Comment (gverni) [reply test/qunit/tests.js]: > It is correct, it's the new ES6 feature:

Wow... never come across that. Agree... it's quite cool...

**Comment by limonte** [comment]
> @limonte what do you think about adding `no-prototype-builtins` which have been [added in `eslint:recommended` in ESLint6](https://eslint.org/docs/user-guide/migrating-to-6.0.0#-eslintrecommended-has-been-updated) and I found quite useful. I couldn't find the same in `eslint-config-standard`

Added :+1:

**Comment by gverni** [comment]
LGTM @limonte if you want to merge it!

**Pull request merged**